docs/monoDrive_home/Camera.md

```diff
@@ -161,76 +227,154 @@ Provides a grayscale camera stream.
        "x": 512.0,
        "y": 512.0
      },
-    "max_distance": 50000.0,
-    "dynamic_range":  50,
     "fov": 60.0,
-    "focal_length": 9.0,
-    "fstop": 1.4,
-    "min_shutter":  0.000500,
-    "max_shutter":  0.001400,
-    "sensor_size":  9.07,
-   "channels" : "rgba"
+    "debug_draw": false,
+    "annotation": false,
+    "include_tags": false,
+    "include_obb": false,
+    "cull_partial_frame": false,
+    "far_plane": 5000.0,
+    "desired_tags": [
+      "dynamic"
+    ]
  }
-
 ]
 ```
 
-<p>&nbsp;</p>
+These configuration values are the same as the RGB camera. Note that the output 
+format of the image is an array the same size as `stream_dimensions` containing 
+32-bit floating point numbers representing depth in centimeters.
 
+## Raw Output
 
-## Configuration
+The total sensor output will be 12 bytes for the monoDrive sensor header plus 
+the total number of bytes for the image defined as:
 
-### Annotation
-All cameras support the annotation feature. The annotation feature will give you bounding boxes for all the dynamic objects in the scene. Add the following json tags to any camera configuration.
+```bash
+  Stream Dimension Width (x)  x  Stream Dimension Height (y)  x  4
```

Comment:
Can we use `*` for multiplication? The `x` is getting confused with the `x` direction, IMO

Reply:
Fixed.